tailwind.config.js

```diff
@@ -67,16 +68,8 @@ module.exports = {
         '2px': '2px',
       },
       colors: {
+        ...themeColors,
```

Comment:
This is shorter than declaring the primary, secondary and accent colors individually